en-US/firefox/new/batm.lang

```diff
@@ -0,0 +1,15 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/new/batm/
```

Comment:
The link is still craig's personal test server. I am fine if we remove it from this PR. Oliver is on vacation, i can wait until craig's work is merged to work on this one.